src/parcel_snoopi.jl

```diff
@@ -297,3 +300,25 @@ function parcel(tinf::AbstractVector{Tuple{Float64,Core.MethodInstance}}; subst=
     end
     return pc
 end
+
+"""
+Search and removes blacklist from pcI
+
+# Examples
+```julia
+blacklist = ["hi","bye"]
+pcI = ["good","bad","hi","bye","no"]
+
+blacklist_remover(blacklist, pcI)
+```
+"""
+function blacklist_remover(blacklist, pcI)
```

Comment:
Please swap the order of arguments and call this `blacklist_remover!`.

https://docs.julialang.org/en/v1.3/manual/style-guide/ (Seem familiar? :stuck_out_tongue_winking_eye: )